golang1.17/Dockerfile

```diff
@@ -0,0 +1,51 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Do not fix the patch level for golang:1.17 to automatically get security fixes.
+FROM golang:1.17-buster
```

Comment:
Should we base on Golang:1.17-bullseye instead?

Reply:
I dont have an opinion on either.

Reply:
It looks to me like buster's LTS EOL is far enough away that we'll have stopped supporting go 1.17 first.  So we can defer the bullseye switch to go 1.18 or 1.19.